install.sh

```diff
@@ -752,10 +766,10 @@ eval set -- "$(escape "$INSTALL_NODE_EXPORTER_EXEC") $(quote "$@")"
   verify_system
   setup_env "$@"
   download_and_verify
+  can_skip_selinux || setup_selinux
```

Comment:
As written before, remove `can_skip_selinux` and move it to `setup_linux`:
```console
...
setup_linux
...
```

Reply:
Done